src/datachain/lib/dc.py

```diff
@@ -1499,6 +1500,7 @@ def from_records(
         Parameters:
             to_insert : records (or a single record) to insert. Each record is
                         a dictionary of signals and theirs values.
+            schema : describes chain signals and their corresponding types
```

Comment:
does the `empty = DataChain.from_records()` still work?

Reply:
(see below in the doc string)

Reply:
Doesn't work, removing it